Tasks/AzureAppServiceManage/operations/KuduServiceUtils.ts

```diff
@@ -53,8 +54,8 @@ export class KuduServiceUtils {
 
         for(var extensionID of extensionList) {
             var siteExtensionDetails = null;
-            if(siteExtensionMap[extensionID]) {
-                siteExtensionDetails = siteExtensionMap[extensionID];
+            if(siteExtensionMap[extensionID] || (extensionID.startsWith('python') && siteExtensionMap[pythonExtensionPrefix + extensionID])) {
```

Comment:
Add some comment on why we are doing this

Reply:
done